libs/validation/src/lib/decorators/number/IsCompositeIdentifier.ts

```diff
@@ -39,8 +39,9 @@ export function IsCompositeIdentifier<TIdType extends string | number>(
         {
             name: isCompositeIdentifier.name,
             validator: {
-                validate: (value, _): boolean =>
-                    isCompositeIdentifier(AllowedTypesEnum, idTypeGuard, value),
+                validate: (value, _): boolean => {
```

Comment:
Reset this change. Probably just wanted to create a closure to hit a breakpoint in the debugger.

Reply:
done